ui-admin/src/study/notifications/EmailTemplateEditor.tsx

```diff
@@ -21,9 +21,12 @@ export default function EmailTemplateEditor({ emailTemplate, updateEmailTemplate
 
   const replacePlaceholders = (html: string) => {
     return html.replaceAll('${siteMediaBaseUrl}', location.origin + getMediaBaseUrl(portalShortcode))
+      // support legacy tempaltes that reference this as siteImageBaseUrl
+      .replaceAll('${siteImageBaseUrl}', location.origin + getMediaBaseUrl(portalShortcode))
   }
   const insertPlaceholders = (html: string) => {
     return html.replaceAll(location.origin + getMediaBaseUrl(portalShortcode), '${siteMediaBaseUrl}')
+      .replaceAll('${siteImageBaseUrl}', location.origin + getMediaBaseUrl(portalShortcode))
```

Comment:
I also just noticed that the new replaceall call is backwards, I think. At least the arguments are reversed compared to the original.